scripts/support-agent.sh

```
@@ -272,48 +314,116 @@ for THREAD_INDEX in $(seq 0 $((PENDING_COUNT - 1))); do
+  # Hand the prompt + bundle to `claude -p` and capture stdout. Claude is
+  # constrained by the prompt + bundle.config.phase to only escalate or
+  # spam-folder in Phase D — the prompt itself enforces this. We don't pipe
+  # the bundle on real stdin because `claude -p` takes the prompt as an
+  # argument and ignores stdin.
+  PROMPT_FILE="$SCRIPT_DIR/support-agent-prompt.md"
+  if [[ ! -f "$PROMPT_FILE" ]]; then
+    log "ERROR: prompt file missing: $PROMPT_FILE"
+    exit 1
+  fi
+  PROMPT_TEXT=$(cat "$PROMPT_FILE")
+  CLAUDE_INPUT=$(printf '%s\n\n## Context bundle for this run\n\n```json\n%s\n```\n' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)